packages/embed-grid/src/index.tsx

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+// Fira fonts are not necessary, but look the best
+import 'fira';
+
+// Need to import the base style sheet for proper styling
+// eslint-disable-next-line import/no-unresolved
+import '@deephaven/components/dist/BaseStyleSheet.css';
+import './index.scss';
+import App from './App';
+
+ReactDOM.render(
+  <React.StrictMode>
```

Comment:
This shows a warning in the browser console - `Warning: Legacy context API has been detected within a strict-mode tree.`

https://stackoverflow.com/questions/61647771/react-reactstrap-warning-legacy-context-api-has-been-detected-within-a-strict